doomrainbow/src/main/java/com/detroitlabs/kyleofori/doomrainbow/RainbowView.java

```diff
@@ -110,9 +133,9 @@ private static void initDefaultForegroundArcPaint() {
 
     /**
      * Aspect ratio of child view, such that
-     *
-     *     w = LAMBDA h
-     *
+     * <p/>
```

Comment:
I didn't know you could do this! Is there any way to see what it looks like on a website or in official documentation?